frontend/src/templates/Field/ChildrenCompound/ChildrenCompoundField.tsx

```diff
@@ -51,6 +51,7 @@ import {
 
 export interface ChildrenCompoundFieldProps extends BaseFieldProps {
```

Comment:
Might be better to create a `MultipleRespondentFieldProps` type in future to contain all MRF related fields.

Reply:
What will the MRF prop type contain?